SU-SWS/ace-sulgryphon
# Lando update

# READY FOR REVIEW

# Summary
- Sets up Lando stuff for localdev on Library and Press sites.
- There isn't a front-end repo yet for Press, so it doesn't include that, but it'll be easy to add.

# Review By (Date)
- 🤷 

# Urgency
- regular

# Steps to Test

1. You must have lando installed to test.  Clone this repo to a convenient place, and follow the instructions in `/docs/lando.md`
2. Verify the localdev spins up correctly.

# Affected Projects or Products
- None

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
